### DISCUSSION TITLE
如何导出部署文件？

### DISCUSSION THREAD

**Discussion by Dysheng** [illacloud/illa-builder#782] [Q&A]
搭建好应用后，没找到如何导出部署文件？

另外，能不能文化自信点，明明是中国人主导的项目，非要默认用英语？不能直接用简体中文吗。
国外的人要看，让他们自己去翻译去啊。

**Comment by karminski** [comment]
您好, 

感谢您的反馈.

关于导出部署文件, 我们暂时不支持导出, 详见 #763 

目前我们的 Discussion 会尽量使用 Discussion 提出者的语言来回复. 使用中文回复中文, 使用英语回复英语. 我们在过去几周内还在 illa-builder 中增加了日语和韩语的支持.  

我们 illasoft 的使命是 "Refactor everyone's work" 这包括所有使用我们开源软件的人. 未来我们会根据社区反馈增加更多的语言支持, 使更多的人使用我们的低代码工具来"重构"他们的工作. 

"No Discrimination Against Persons or Groups" ( [From  Debian Free Software Guidelines](https://www.debian.org/social_contract) ) 我们相信, 只有平等的对待每一位用户, 才能促成我们的愿景. 这也是开源的精神所在. 

欢迎您提交 PullRequests 和参与 Discussion, 我们相信有了您的贡献, illacloud 社区会蓬勃发展, 这将惠及每个使用 illa 的用户. 

再次感谢您的意见.

--- 

Hello, 

Thanks for your feedback.

At present, our Discussion will try to use the language of the Discussion proposer to reply. Use Chinese to reply to Chinese, and English to reply to English. We have also added Japanese and Korean support in illa-builder in the past few weeks.

Our mission at illasoft is "Refactor everyone's work", which includes everyone who uses our open-source software. In the future, we will add more language support based on community feedback, so that more people can use our low-code tools to "refactor" their work. 

"No Discrimination Against Persons or Groups" ( [From Debian Free Software Guidelines](https://www.debian.org/social_contract) ) We believe that only by treating every user equally can we achieve our vision. This is also The spirit of open source.

You are welcome to submit PullRequests and participate in Discussion, we believe that with your contributions, the illacloud community will flourish, which will benefit every user who uses illa.

Thanks again for your discussion.
